Makefile

```diff
@@ -93,6 +93,31 @@ vet: ## Run go vet against code.
 test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
 
+
+OPERATOR_NAMESPACE ?= openshift-storage
+TOPOLVM_CSI_IMAGE ?= quay.io/topolvm/topolvm:0.10.3
+CSI_REGISTRAR_IMAGE ?= k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.3.0
+CSI_PROVISIONER_IMAGE ?= k8s.gcr.io/sig-storage/csi-provisioner:v3.0.0
+CSI_LIVENESSPROBE_IMAGE ?= k8s.gcr.io/sig-storage/livenessprobe:v2.5.0
+CSI_RESIZER_IMAGE ?= k8s.gcr.io/sig-storage/csi-resizer:v1.3.0
+VGMANAGER_IMAGE ?= quay.io/ocs-dev/vgmanager:latest
+
+
+define MANAGER_ENV_VARS
+OPERATOR_NAMESPACE=$(OPERATOR_NAMESPACE)
+TOPOLVM_CSI_IMAGE=$(TOPOLVM_CSI_IMAGE)
+CSI_REGISTRAR_IMAGE=$(CSI_REGISTRAR_IMAGE)
+CSI_PROVISIONER_IMAGE=$(CSI_PROVISIONER_IMAGE)
+CSI_LIVENESSPROBE_IMAGE=$(CSI_LIVENESSPROBE_IMAGE)
+CSI_RESIZER_IMAGE=$(CSI_RESIZER_IMAGE)
+VGMANAGER_IMAGE=$(VGMANAGER_IMAGE)
+endef
+export MANAGER_ENV_VARS
+
+update-mgr-env: ## Feed env variables to the manager configmap
```

Comment:
better have this target as well in `deploy` as it's changing the file used in kustomize?

Reply:
I didn't want to do that for the development targets. `make update-mgr-env` always be called explicitly if required before a `make deploy`.